src/StackExchange.Redis/Interfaces/IDatabase.cs

```diff
@@ -516,6 +516,15 @@ public interface IDatabase : IRedis, IDatabaseAsync
         /// <remarks>https://redis.io/commands/dump</remarks>
         byte[]? KeyDump(RedisKey key, CommandFlags flags = CommandFlags.None);
 
+        /// <summary>
+        /// Returns the internal encoding for the Redis object stored at <paramref name="key"/>.
+        /// </summary>
+        /// <param name="key">The key to dump.</param>
+        /// <param name="flags">The flags to use for this operation.</param>
+        /// <returns>The serialized value.</returns>
+        /// <remarks>https://redis.io/commands/object-encoding</remarks>
+        RedisValue KeyEncoding(RedisKey key, CommandFlags flags = CommandFlags.None);
```

Comment:
I think this should be a `string?` based on the API - it'll only ever be a bulk string or null right?

Reply:
yup